tests/test_hello_world.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from hello_app import app
+
+
+def test_hello_world_handler_correct_header_input():
+    """
+    test the hello_world handler to conform its 
+    returning the expected values based on specific input
```

Comment:
"Confirm that a 200 status code is returned where a "accept: application/json+ld" header is provided."